src/System.Security.Cryptography.Primitives/tests/CryptographicException.cs

```diff
@@ -20,5 +21,44 @@ public static void Ctor()
             Assert.Same(inner, new CryptographicException(message, inner).InnerException);
             Assert.Equal(message, new CryptographicException(message, inner).Message);
         }
+
+        [Theory]
+        [InlineData(0x00000000, "The operation completed successfully")]
+        [InlineData(0x80090013, "Invalid provider specified")]
+        [InlineData(0x80090016, "Keyset does not exist")]
```

Comment:
Yeah, I'm hopeful that these messages haven't been updated since the dawn of time, and will continue to be in the "we already have a message for this" camp.